src/app/client/src/app/modules/learn/components/course-consumption/course-player/course-player.component.ts

```diff
@@ -163,6 +166,11 @@ export class CoursePlayerComponent implements OnInit, OnDestroy {
         }
       })).subscribe((response: any) => {
         this.courseHierarchy = response.courseHierarchy;
+        const contentCredits = _.get(this.courseHierarchy, 'content-credits');
+        if (contentCredits) {
+          this.contributionsLength = contentCredits.length;
+        }
+        this.contributions = _.map(contentCredits, 'name').toString();
```

Comment:
`_.map(contentCredits, 'name').toString();` should be `_.toString(_.map(contentCredits, 'name'));` for better error handling